src/sidebar.jsx

```diff
@@ -335,6 +339,17 @@ const DropdownWithKebab = ({
             },
             title: cockpit.format(_("Rename $0"), selected[0]?.type || "directory")
         },
+        ...(selected.length === 1 && selected[0].type === "file" && allowedtoRead(selected[0], user))
+
+            ? [
+                { type: "divider" },
+                {
+                    id: "download-item",
+                    onClick: () => downloadFile(currentPath, selected[0]),
+                    title: _("Download file")
```

Comment:
Why does this appear both here and in app.jsx? Does this duplicate the menu for some reason?

Reply:
Yes, they are different components and duplicated

Reply:
Ack.